### DISCUSSION TITLE
Grouping supersets, self-resetting timers

### DISCUSSION THREAD

**Discussion by oddomatik** [astashov/liftosaur#154] [Ideas]
I know the suggested way to do a superset is to just work vertically, but it would be nice to have it displayed in a way that makes it clear it's a superset. This would be best for sharing workouts that include them, allowing those unfamiliar with the methodology to immediately understand the workout without reading comments or whatever.

For example, a minimized form of each exercise and sets/reps - that has a clickable header to show form and details if desired - that is grouped within a larger container. I may not want to have to click a button between each exercise with a superset, but instead I want to click one (or a group of some that are close together) between supersets.

Supersetting timers might also look a little different than normal. A lot of workouts use a circuit-type format, where one should do a plank for 30s, then go into AMRAP for a given time of a weight exercise, etc... This functionality is ALMOST there in the app, but I want for these exercises the chime in my ear that indicates the transition, without having to go click a button in the app. Between supersets I can go and mark the exercise complete and indicate my progress. We might also add in a transition time, and two different sounds for end set/start next set.

**Comment by yrinkel** [comment]
Please check if your idea here is not redundant with the previously opened discussion:
https://github.com/astashov/liftosaur/discussions/151#discussion-7118939

In case it is the same, please close this one to avoid redundancies.

**Reply by oddomatik** [reply]
My apologies, only three topics were displayed when I checked "discussions" page and I thought that was exhaustive.
